src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/action/SetUpAction.kt

```diff
@@ -11,6 +11,7 @@ class SetUpAction(
     override fun run() {
         meter.measure(SET_UP) {
             jira.configureRichTextEditor().disable()
+            jira.configureBackupPolicy().disable()
```

Comment:
This is quite a destructive operation. It actually `delete()`s backup services.
This `SetUpAction` is used by default in most APIs, including app vendors and customers. I believe we shouldn't destroy Jira information by default. It should be possible to opt-in to, e.g. by exposing this new API and letting users customize their scenarios.